app/views/setupquestions/annualallowance/PIAAboveAnnualAllowanceIn2023View.scala.html

```diff
@@ -0,0 +1,46 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@this(
+    layout: templates.Layout,
+    formHelper: FormWithCSRF,
+    govukErrorSummary: GovukErrorSummary,
+    govukRadios: GovukRadios,
+    govukButton: GovukButton
+)
+
+@(form: Form[_], mode: Mode)(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("pIAAboveAnnualAllowanceIn2023.title"))) {
+
+    @formHelper(action = controllers.setupquestions.annualallowance.routes.PIAAboveAnnualAllowanceIn2023Controller.onSubmit(mode), 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form))
+        }
+
+        @govukRadios(
+            RadiosViewModel.yesNo(
+                field = form("value"),
+                legend = LegendViewModel(messages("pIAAboveAnnualAllowanceIn2023.heading")).asPageHeading()
+            )
+        )
+
+        @govukButton(
+            ButtonViewModel(messages("site.continue"))
+        )
```

Comment:
Needs our save and return button component

Reply:
Done

Reply:
See here for our custom govukbutton

https://github.com/hmrc/calculate-public-pension-adjustment-frontend/blob/8d089b2ba0a1c7aa5eded6ca7f3b9173282cc7d8/app/views/annualallowance/taxyear/AddAnotherSchemeView.scala.html#L50C7-L50C26